.github/workflows/rust.yml

```diff
@@ -46,4 +46,7 @@ jobs:
         uses: dtolnay/rust-toolchain@stable
 
       - name: cargo test
```

Comment:
I suggest a comment to explain why the test all below isn't sufficient, at a glance it looks like a mistake

Reply:
this is a normal pattern, because you want to know that you build with the default features as well as with all features? We do it in a bunch of projects. Happy to add a comment, though..